### PR TITLE
JCRVLT-429 switch to bnd-maven-plugin

### PR DESCRIPTION
**Pull request by kwin**

### CODE REVIEW

File: vault-core/pom.xml [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd ">
     <modelVersion>4.0.0</modelVersion>
     <!-- ====================================================================== -->
-    <!-- P A R E N T  P R O J E C T  D E S C R I P T I O N                      -->
+    <!-- P A R E N T P R O J E C T D E S C R I P T I O N -->
```
Comment (tripodsan) [inline vault-core/pom.xml:21]: nitpick: ether keep formatting or choose a different commenting style. but by removing all the spaces, it becomes unreadable.

File: vault-rcp/src/main/java/org/apache/jackrabbit/vault/rcp/RcpTaskManager.java [hunk 2]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.jackrabbit.vault.rcp.impl;
+package org.apache.jackrabbit.vault.rcp;
```
Comment (tripodsan) [inline vault-rcp/src/main/java/org/apache/jackrabbit/vault/rcp/RcpTaskManager.java:17]: why has this changed?

Comment (kwin) [reply vault-rcp/src/main/java/org/apache/jackrabbit/vault/rcp/RcpTaskManager.java]: I will take the changes from rcp out -> tracked now in https://issues.apache.org/jira/browse/JCRVLT-438

**Comment by kwin** [comment]
Merged in https://github.com/apache/jackrabbit-filevault/commit/a26e30888d9077f9451d71dec67024b2de859f3f

**Pull request closed without merging**